qiskit/providers/backend.py

```diff
@@ -229,3 +241,445 @@ class can handle either situation.
+                a list is specified the output will be a list with the
+                T1 time for the specified qubits in the same order.
+
+        Returns:
+            t1: the T1 time for the specified qubit(s) in seconds.
+
+        Raises:
+            NotImplementedError: if the backend doesn't support querying the
+                qubit properties for a qubit
+            ValueError: if T1 is not defined for the specified qubit
+        """
+        props = self.qubit_properties(qubits)
+        if isinstance(props, QubitProperties):
+            if props.t1 is None:
+                raise ValueError(f"T1 is not defined for qubit {qubits}")
+            return props.t1
+        out_list = []
+        for index, prop in enumerate(props):
+            if prop.t1 is None:
+                raise ValueError(f"T1 is not defined for qubit {qubits[index]}")
+            out_list.append(prop.t1)
+        return np.array(out_list)
+
+    def t2(self, qubits: Union[int, List[int]]) -> Union[float, np.array]:
+        """Return the T2 time of a given qubit
+
+        Args:
+            qubits: The qubit index or indices to get the T2 time for. If
+                a list is specified the output will be a list with the
+                T2 time for the specified qubits in the same order.
+
+        Returns:
+            t2: the T2 time for the specified qubit(s) in seconds
+
+        Raises:
+            NotImplementedError: if the backend doesn't support querying the
+                qubit properties for a qubit
+            ValueError: if T2 is not defined for the specified qubit
+        """
+        props = self.qubit_properties(qubits)
+        if isinstance(props, QubitProperties):
+            if props.t2 is None:
+                raise ValueError(f"T2 is not defined for qubit {qubits}")
+            return props.t1
+        out_list = []
+        for index, prop in enumerate(props):
+            if prop.t2 is None:
+                raise ValueError(f"T2 is not defined for qubit {qubits[index]}")
+            out_list.append(prop.t2)
+        return np.array(out_list)
+
+    @property
+    def dt(self) -> Union[float, None]:
+        """Return the system time resolution of input signals
+
+        This is required to be implemented if the backend supports Pulse
+        scheduling.
+
+        Returns:
+            dt: The input signal timestep in seconds. If the backend doesn't
+            define ``dt`` ``None`` will be returned
+        """
+        return self.target.dt
+
+    @property
+    def dtm(self) -> float:
+        """Return the system time resolution of output signals
+
+        Returns:
+            dtm: The output signal timestep in seconds.
+
+        Raises:
+            NotImplementedError: if the backend doesn't support querying the
+                output signal timestep
+        """
+        raise NotImplementedError
+
+    @property
+    def meas_map(self) -> List[List[int]]:
```

Comment:
Is this needed for base class too?

Reply:
It is because it's a required field for the pulse scheduler: https://github.com/Qiskit/qiskit-terra/blob/main/qiskit/compiler/scheduler.py#L80

That being said a backend doesn't need to implement it if they don't when the pulse scheduler is called it will raise a `NotImplementedError` to indicate the backend isn't a valid target for pulse scheduling. This is just defining an optional standard interface to extract this information from a backend

Reply:
Probably it's good timing to rewrite scheduler macro. We are used to have this restriction before (due to hardware trigger) but this is no longer needed, at least for IBM backends. I'm not sure this is still important for other providers.